openstates/in/bills.py

```diff
@@ -36,6 +39,7 @@ def check_vote_counts(vote):
         pass
```

Comment:
What if we did a `self.warning()` instead of `pass`ing on these?
Also, `check_vote_counts` can be replaced with `vote.validate()`, which checks for this exact issue. I mentally facepalmed when I found that out, since I've written at least three or four vote validators myself, haha.